File: app/src/main/java/com/imangazalievm/bubbble/presentation/utils/DebugUtils.java
package com.imangazalievm.bubbble.presentation.utils;

import android.widget.Toast;

import com.imangazalievm.bubbble.BubbbleApplication;
import com.imangazalievm.bubbble.BuildConfig;

public class DebugUtils {

    public static void showDebugErrorMessage(Throwable throwable) {
        if (BuildConfig.DEBUG) {
            throwable.printStackTrace();
            Toast.makeText(BubbbleApplication.getInstance(), throwable.getMessage(), Toast.LENGTH_SHORT).show();
        }
    }

}

File: app/src/main/java/com/imangazalievm/bubbble/presentation/global/utils/DebugUtils.kt
package com.imangazalievm.bubbble.presentation.global.utils

import android.widget.Toast
import com.imangazalievm.bubbble.BubbbleApplication
import com.imangazalievm.bubbble.BuildConfig

object DebugUtils {

    @JvmStatic
    fun showDebugErrorMessage(throwable: Throwable) {
        if (BuildConfig.DEBUG) {
            throwable.printStackTrace()
            Toast.makeText(BubbbleApplication.instance, throwable.message, Toast.LENGTH_SHORT)
                .show()
        }
    }

}